website/inertia/pages/tickets.tsx

```diff
@@ -0,0 +1,39 @@
+'use client'
+
+import Ticket from '#models/ticket'
```

Comment:
If you need to use types from models that are defined in the backend, always use `import type`.